log4j-core/src/test/java/org/apache/logging/log4j/core/LateConfigTest.java

```diff
@@ -26,6 +26,7 @@
 import org.apache.logging.log4j.core.config.DefaultConfiguration;
 import org.apache.logging.log4j.core.config.xml.XmlConfiguration;
 import org.apache.logging.log4j.status.StatusLogger;
+
```

Comment:
We don't require whitespace here. We don't disallow it either, but not sure what the point is.